modules/Movecraft/src/main/resources/types/Ship.craft

```diff
@@ -96,9 +96,13 @@ forbiddenBlocks:
 
 canCruise: true
 cruiseSkipBlocks: 4
+fuelBurnRate: 0.0
+cruiseSpeed: 1.0
 canFly: false
 sinkPercent: 99.0
 overallSinkPercent: 60.0
+explodeOnCrash: 0.0
+sinkSpeed: 3.0
```

Comment:
The default options for these work fine, I don't understand the purpose of this PR.  The plugin comes with default types and you can modify them on your server without a PR like this.